tools/qmlformat.py

```diff
@@ -28,7 +28,16 @@ def find_qt_version():
 
 
 def main(argv=None):
-    qmlformat_executable = shutil.which("qmlformat")
+    # First look up at the most common location for QT6 which is
+    # usually not in PATH
+    if sys.platform != "win32":
+        unix_paths = ("/usr/lib/qt6/bin", "/usr/lib64/qt6/bin")
+        for path in unix_paths:
+            qmlformat_executable = shutil.which("qmlformat", path=path)
```

Comment:
This loop is unnecessary, `path` is equivalent to `$PATH`, i.e. may contain multiple paths separated by `os.pathsep` (`:` on unix, `;` on windows).

What we do need a loop for is both executable name variants:

```suggestion
        path = os.pathsep.join(unix_paths)
        for executable_name in ("qmlformat-qt6", "qmlformat"):
            qmlformat_executable = shutil.which(executable_name, path=path)
```